sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -2460,9 +2460,8 @@ static void clearCreatedTables() {
     private static Set<String> createdTables =
         Collections.newSetFromMap(new ConcurrentHashMap<String, Boolean>());
 
-    /** Tracks bytes written, exposed as "ByteCount" Counter. */
-    private Aggregator<Long, Long> byteCountAggregator =
-        createAggregator("ByteCount", Sum.ofLongs());
```

Comment:
Can you also delete the `@SystemDoFnInternal` above, as that was only added to support the Aggregator?